ImprovedAnvil/lib/scripts/00_compile.tph

```diff
@@ -773,6 +773,7 @@ COMPILE ~ImprovedAnvil/scripts/npc/MORSWORD.baf~
 COMPILE ~ImprovedAnvil/scripts/npc/PPJON.baf~
 COMPILE ~ImprovedAnvil/scripts/npc/S!xp.baf~
 COMPILE ~ImprovedAnvil/scripts/npc/VALYGX.baf~
+COMPILE ~ImprovedAnvil/scripts/npc/OHHPHREY.baf~
```

Comment:
you don't need to compile the script separately when you are EXTENDing. The updated version gets compiled automatically. What you're doing here is basically replacing the original script with the new one AND THEN you're basically extending on top of it another identically copy (since you've just re-compiled it).

Reply:
interesting - so to summarize this line can be deleted?

Reply:
Yes